src/main/java/edu/cornell/kfs/sys/service/impl/ReportWriterTextServiceImpl.java

```diff
@@ -66,6 +70,36 @@ public void initialize() {
 		this.writeHeader(title);
 	}
 
+	@Override
+    public void initalize(String dataFileNamePrefix) {
```

Comment:
Some of the formatting is a bit off. If the newly-added methods are only meant to contain spaces, please check to make sure their first and last lines have been converted from tabs to spaces accordingly.